src/components/TextInput/style.scss

```diff
@@ -4,7 +4,7 @@
     border-radius: 2px;
     box-sizing: border-box;
     margin: $element-margin;
-    min-height: 24px;
+    height: 24px;
```

Comment:
Can we keep the min-height as well? Because it will affect the flexbox

Reply:
Sure 👍